UnitTests/GitUI.Tests/Script/ScriptOptionsParserTests.cs

```diff
@@ -354,6 +354,40 @@ public void ParseScriptArguments_resolve_RepoName()
             result.Should().Be(dirName);
         }
 
+        [Test]
+        public void ParseScriptArguments_resolve_cHead_with_checkedout_branch_name()
+        {
+            var option = "cHead";
```

Comment:
use of var
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0007-ide0008

Reply:
In the gist:
- don't use `var` for the intrinsic types, and
- don't use `var` where the type isn't easily inferable from the context (e.g., `var thing = DoStuff()` - what's the type of `thing`?).